packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/PropertyProvider.cs

```diff
@@ -19,18 +19,16 @@ public class PropertyProvider
         private VariableExpression? _variable;
         private Lazy<ParameterProvider> _parameter;
 
-        public FormattableString Description { get; }
-        public XmlDocSummaryStatement XmlDocSummary { get; }
+        public FormattableString? Description { get; }
```

Comment:
Should we have this as a property?  Why not change the callers to simply modify the XmlDocProvider?

Reply:
Will do a follow-up pass where we address the `Description` property consistently across providers. I agree we can remove them.